src/makers/kerlink/hotspots.ts

```diff
@@ -0,0 +1,73 @@
+import HotspotIcon_iFemto from '@ifemto.svg'
```

Comment:
There imports should look like this
```ts
import HotspotIcon_iFemto from './ifemto.svg'
import HotspotIcon_iFemtoEvo from './ifemto_evo.svg'
import HotspotIcon_iStation from './istation.svg'
```

Reply:
right, my bad, I had written that code when images where in the assets folder